live-examples/js-examples/typedarray-some.html

```diff
@@ -0,0 +1,12 @@
+<pre>
+<code id="static-js" class="language-js">function isNegative(element, index, array) {
+  return element < 0;
+}
+
+var int8 = new Int8Array([-10, 20, -30, 40, -50]);
+
+// test every element - at least one must pass
+console.log(int8.some(isNegative));
+// expected output: true
+</code>
+</pre>
```

Comment:
Nit: Again maybe worth considering an example that will return false:

```
function isNegative(element, index, array) {
  return element < 0;
}

var int8 = new Int8Array([-10, 20, -30, 40, -50]);
var positives = new Int8Array([10, 20, 30, 40, 50]);

console.log(int8.some(isNegative));
// expected output: true
console.log(positives.some(isNegative));
// expected output: false
```

... but be careful to keep to 12 lines :). I would omit the comment at line 8.